src/encoded/types/gene.py

```diff
@@ -0,0 +1,81 @@
+from snovault import (
+    calculated_property,
+    collection,
+    load_schema,
+)
+from .base import (
+    SharedItem,
+    paths_filtered_by_status,
+)
+from pyramid.traversal import (
+    find_root,
+)
+
+
+@collection(
+    name='genes',
+    unique_key='gene:name',
+    properties={
+        'title': 'Genes',
+        'description': 'Listing of ENCODE genes',
+    })
+class Gene(SharedItem):
+    item_type = 'gene'
+    schema = load_schema('encoded:schemas/gene.json')
+    embedded = ['organism']
+    rev = {
+        'targets': ('Target', 'gene')
+    }
+
+    def unique_keys(self, properties):
+        keys = super(Gene, self).unique_keys(properties)
+        keys.setdefault('gene:name', []).append(self._name(properties))
+        return keys
+
+    @calculated_property(schema={
+        "title": "Name",
+        "type": "string",
+    })
+    def name(self):
+        return self.__name__
+
+    @calculated_property(schema={
+        "title": "Title",
+        "type": "string",
+    })
+    def title(self, request, organism, symbol):
+        organism_props = request.embed(organism, '@@object')
+        return u'{} ({})'.format(symbol, organism_props['scientific_name'])
+
+    @calculated_property(schema={
+        "description": "Targets targeting this gene.",
```

Comment:
I feel like this is a little jargon-y.  But Antibody targets isn't 100% accurate either, is it?

Reply:
Corrected as "List of associated targets." in ENCD-3997-modification-1 commit.